client/src/lib/ErrorHandling.js

```diff
@@ -0,0 +1,15 @@
+import { CircularProgress } from '@mui/material';
+
+export function handleLoading(isLoading) {
```

Comment:
I don't view these functions as "handling" anything. Instead they are "showing" a loading or an error message. What about naming them `showLoading` and `showError` instead?